src/action/payment.js

```diff
@@ -170,6 +170,11 @@ class PaymentAction {
    * @return {Promise<undefined>}
    */
   async payLightning() {
+    let failed = false;
+    const timeout = setTimeout(() => {
```

Comment:
How can I test this block manually? In my local simnet environment I'm always getting an error from the stream `data.payment_error` and it just shows the notification.

Reply:
I tested by manually putting a sleep before the failed check, forcing a timeout. Not the most robust :grimacing:

Reply:
@valentinewallace well, that's one way of doing it :D.